src/modules/jcms_notifications/drush/jcms_notifications.drush.inc

```diff
@@ -36,6 +36,20 @@ function jcms_notifications_drush_command() {
       'drush article-import-all --skip-updates' => 'Import all articles from Lax, but skip over articles that exist already, and return a message when finished.',
     ],
   ];
+  $items['send-notifications'] = [
+    'description' => 'Gets notifications from the database and send them to SNS.',
+    'options' => [
+      'iterations' => 'Limit on the number of iterations made then terminate.',
+      'sleep' => 'The amount of time in seconds to sleep before iterating over the notifications table again (defaults to 30 seconds).',
+    ],
+    'drupal dependencies' => ['jcms_notifications'],
+    'aliases' => ['sendn'],
+    'examples' => [
```

Comment:
@giorgiosironi These examples show how the long running process can be changed to a fixed number of iterations and also that the sleep time can be modified. The default is `drush send-notifications`.